tasks/sign-base64-blob/README.md

```diff
@@ -28,3 +28,5 @@ data:
 
 ## Changes in 1.0.1
 - Now the task decodes the payload from base64 before creating the `.sig` file
+## Changes in 1.0.2
+- The `.sig` file is not a clear ASCII GPG signature anymore but a not ASCII armored GPG binary.
```

Comment:
It's not clear to me what's happening. At first I thought the payload is non-ascii and we want to make ascii from it. But it's the opposite, right? Just curious, why is this change happening? In any case, I would suggest to reword this slightly to make it a bit clearer, e.g.:
```
- Save the `.sig` file as a non-ASCII armored GPG binary instead of clear ASCII GPG signature
```

Reply:
Right from the outset, our objective was to generate a GPG signature in non-ASCII binary format in accordance with the specified requirement. However, upon testing the whole workflow, we saw that the resulting GPG .sig file is in clear ASCII format, contrary to the intended. Hence the PR. Will reword with your suggestion.

Reply:
OK, thanks for the explanation.